File: Code/App/separateDB/Ward/ViewModels/WardPatientDeclarationViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using RepositoryClasses.Models;

namespace Ward.ViewModels
{
    public class WardPatientDeclarationViewModel
    {
        public PatientsModel PatientInfo { get; set; }
        public WardPatientCurrentDieseaseViewModel CurrentDiesease { get; set; }
    }

}

File: Code/App/separateDB/Ward/Scripts/App/Models/WardPatientDeclarationViewModel.js
function WardPatientDeclarationViewModel(data) {
    var self = this;

    this.PatientInfo = new PatientsModel(data.PatientInfo);

    this.DieseaseLogs = ko.observableArray([]);
    this.DieseaseLogs.push(new DieseaseLogs(data));
};